common/thorhelper/thorsoapcall.cpp

```diff
@@ -2504,13 +2524,15 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
                 {
                     if (master->timeLimitExceeded)
                     {
+                        clientSpan->recordError("time_limit_exceeded");
```

Comment:
"time_limit_exceeded" is likely only good enough to denote the type of error encountered, but do we want to supply full log messages? or maybe something in between.
This is likely going to be handled differently on case-by-case basis